components/playlist/browser/resources/components/player.tsx

```diff
@@ -28,19 +28,35 @@ import {
 const StyledVideo = styled.video`
   width: 100vw;
   aspect-ratio: 16 / 9;
-  margin-bottom: 8px;
+  margin-bottom: var(--spacing-between-video-and-controls, 8px);
 `
 
-const PlayerContainer = styled.div`
+const PlayerContainer = styled.div<{ backgroundUrl?: string }>`
+  --spacing-between-video-and-controls: 8px;
   width: 100vw;
   height: 100vh;
   display: flex;
+  position: relative;
   flex-direction: column-reverse;
   overflow: hidden;
   user-select: none; // In order to drag-and-drop on the seeker works well.
+
+  // Put blurred thumbnail underneath other UI controls
```

Comment:
Yeah, it's very subtle effect so you might not notice it. Player's control area background has 0.97 opacity, so this blurry thumbnail is visible through it, but it's really subtle. @aguscruiz and I was talking about making it more transparent so that the effect can be more noticeable.

Reply:
When we lower the opacity like 0.4, it'll look like

<img width="608" alt="image" src="https://github.com/brave/brave-core/assets/5474642/f1d3861b-20d2-4441-94d3-e69754250319">

Reply:
got it. I'm ok with any transparency :)